### PR TITLE
Add JNAS tts1 recipe

### PR DESCRIPTION
**Pull request by kan-bayashi**
This PR add JNAS tts1 recipe.
(Should be merged after #1263)

- [x] make a recipe
- [x] update `egs/README.md`
- [x] add `egs/jnas/README.md`
- [x] upload pretrained model and add `RESULTS.md`

This is the 10-th tts1 recipe!

### CODE REVIEW

**Comment by kan-bayashi** [comment]
Almost done. Please wait for the training results.

**Comment by kan-bayashi** [comment]
ETA
```
1           500         321.531       1.26002                           1.21379                                1.3346                                 0.0462309                                0.999142                                           0.996953
     total [..................................................]  0.32%
this epoch [##############################....................] 61.03%
       500 iter, 1 epoch / 500 epochs
    1.5057 iters/sec. Estimated time to finish: 1 day, 4:32:55.321005.
1           600         376.221       1.18099                           1.13773                                1.19396                                0.0432608                                0.998992                                           0.99616
     total [..................................................]  0.39%
this epoch [##############################################....] 93.24%
       600 iter, 1 epoch / 500 epochs
    1.5608 iters/sec. Estimated time to finish: 1 day, 3:31:22.974794.
2           700         477.022       1.12969     0.964547              1.08959       0.932585                 1.1271        0.749351                 0.040092       0.0319619                 0.998879            0.998901                       0.995395            0.995553
     total [..................................................]  0.45%
this epoch [############......................................] 25.44%
       700 iter, 2 epoch / 500 epochs
    1.4247 iters/sec. Estimated time to finish: 1 day, 6:08:00.403148.
```

**Comment by kan-bayashi** [comment]
I generated sample using multi-speaker transformer @ 300 epochs.
It looks working well.
Here is the sample.
https://drive.google.com/open?id=1aIPLNi38TsrbZjVtq-XeH-o7Pt2TAfhy
Now we can train Japanese multi-speaker Transformer!
(I think Tacotron2 can also be trained, but not yet tried)

**Comment by kan-bayashi** [comment]
Finished.
I think the quality is better than libriTTS ones.
```
# v.0.5.3 / Initial Transformer

- FTT in points: 1024
- Shift in points: 256
- Frequency limit: 80-7600
- Fast-GL 64 iters

## Environments
- date: `Tue Oct 15 00:29:02 JST 2019`
- python version: `3.7.3 (default, Mar 27 2019, 22:11:17)  [GCC 7.3.0]`
- espnet version: `espnet 0.5.3`
- chainer version: `chainer 6.0.0`
- pytorch version: `pytorch 1.0.1.post2`
- Git hash: `87a0d491358dea338dd02b0f7c8a5cec4c2f8644`
  - Commit date: `Mon Oct 14 13:16:24 2019 +0900`

## Models

- model link: https://drive.google.com/open?id=1fmcr_E_A4bZ2rap4_Aiksg_utlOhXfUJ
- training config file: `conf/train_pytorch_transformer+spkemb.yaml`
- decoding config file: `conf/decode.yaml`
- cmvn file: `data/phn_train_nodev_trim/cmvn.ark`
- e2e file: `exp/phn_train_nodev_trim_pytorch_train_pytorch_transformer+spkemb/results/model.last1.avg.best`
- e2e JSON file: `exp/phn_train_nodev_trim_pytorch_train_pytorch_transformer+spkemb/results/model.json`
- dict file: `data/lang_1char/phn_train_nodev_trim_units.txt`

## Samples

https://drive.google.com/open?id=1fFMQDF6NV5Ysz48QLFYE8fEvbAxCsMBw
```

File: egs/libritts/tts1/conf/train_pytorch_transformer+spkemb.yaml [hunk 1]
```diff
@@ -1,7 +1,5 @@
-# This configuration reuqires 3 gpus in the case of each gpu memory = 12GB, and it takes 2~3 days.
-# If you cannot use 3 gpus, it is better to make batch-bins small, increase accum_grad and epochs.
-# This model uses speaker emebdding with add integration, leading fast attention learning compared
-# to concat integration wihtout guided attention loss.
+# This is the basic transformer training settting with speaker embdding.
```
Comment (r9y9) [inline egs/libritts/tts1/conf/train_pytorch_transformer+spkemb.yaml]: settting -> setting
embdding -> embedding

Comment (kan-bayashi) [reply egs/libritts/tts1/conf/train_pytorch_transformer+spkemb.yaml]: Thanks!

**Pull request merged**